lib/OpenQA/Git/GitServerOutage.pm

```diff
@@ -0,0 +1,39 @@
+
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+package OpenQA::Git::GitServerOutage;
```

Comment:
The package name makes it sound like this represents a concrete outage but it is rather for tracking availability in general. I'd also remove the `Git…` prefix because it is within `OpenQA::Git` anyway.

```suggestion
package OpenQA::Git::ServerAvailability;
```